pkg/services/ngalert/store/alertmanager.go

```diff
@@ -78,10 +74,23 @@ func (st DBstore) SaveAlertmanagerConfigurationWithCallback(ctx context.Context,
 			ConfigurationVersion:      cmd.ConfigurationVersion,
 			Default:                   cmd.Default,
 			OrgID:                     cmd.OrgID,
+			CreatedAt:                 time.Now().Unix(),
```

Comment:
why do we need to do this? This column _should_ be automanaged by xorm?

Reply:
Will address in the future as discussed offline